template/docroot/sites/default/settings/base.settings.php

```diff
@@ -27,7 +27,7 @@
 $is_ah_prod_env = ($ah_env == 'prod' || $ah_env == '01live');
 $is_ah_stage_env = ($ah_env == 'test' || $ah_env == '01test');
 $is_ah_dev_cloud = (!empty($_SERVER['HTTP_HOST']) && strstr($_SERVER['HTTP_HOST'], 'devcloud'));
-$is_ah_dev_env = (preg_match('/^dev[0-9]*$/', $ah_env) == TRUE);
```

Comment:
Heh, I knew you'd ask that... the answer is I have no clue. I've only seen `01`, and I know that ACSF can't have multiple dev environments, so I don't know how it would be anything else.

I'll try to find out.